spyder/config/main.py

```diff
@@ -400,13 +400,14 @@
               '_/switch to variable_explorer': "Ctrl+Shift+V",
               '_/switch to find_in_files': "Ctrl+Shift+F",
               '_/switch to explorer': "Ctrl+Shift+X",
-              '_/switch to plots': "Ctrl+Shift+G",
+              '_/switch to plots': "Shift+F3" if MAC else "Ctrl+Shift+G",
```

Comment:
Could you suggest another `Ctrl+Shift` shortcut? That way we would preserve our convention of using `Ctrl+Shift` shortcuts for plugins.

Reply:
The following alphabetical characters are available in combination with "Ctrl+Shift+": (K, J, Y, Q, A, N, M).
Do you have a preference? I recommend "J" since it has the least overlap with other meta-key combinations so one is less likely to accidentally engage the wrong action, e.g. quit the application with "Q".

Given the convention for "Ctrl+Shift" for switching plugins, do you also want consistency across platforms as much as possible? I.e. should we always use "Ctrl+Shift+J", not just for macOS?

Reply:
> Do you have a preference?

Nop

> I recommend "J" since it has the least overlap with other meta-key combinations so one is less likely to accidentally engage the wrong action, e.g. quit the application with "Q"

Ok, good idea.

> Given the convention for "Ctrl+Shift" for switching plugins, do you also want consistency across platforms as much as possible?

That's a good idea too, but I think by now people that use `Ctrl+Shift+G` should be quite accustomed to that shortcut, so I'd prefer to not break their workflows on Windows and Linux (at least on Mac there's a strong reason to change it). But if users let us know in new issues that they prefer consistency over familiarity, then we can change it to `Ctrl+Shift+J` in a later release.